deploy/sre-prometheus/ocm-agent/100-ocm-agent.PrometheusRule.yaml

```diff
@@ -10,6 +10,14 @@ spec:
   groups:
   - name: sre-managed-notification-alerts
     rules:
+      - alert: KubePersistentVolumeFillingUpSRE
```

Comment:
is this not correctly indented? the build is complaining about this line :thinking:

Reply:
This is indented as in the next rule.
Yeah, maybe I could indent one space more.

Reply:
I thought you were talking about the comment. Indeed bad indent.